deps/v8/include/v8.h

```diff
@@ -3194,15 +3195,13 @@ class FunctionCallbackInfo {
   static const int kDataIndex = 4;
   static const int kCalleeIndex = 5;
   static const int kContextSaveIndex = 6;
+  static const int kNewTargetIndex = 7;
 
   V8_INLINE FunctionCallbackInfo(internal::Object** implicit_args,
-                   internal::Object** values,
-                   int length,
-                   bool is_construct_call);
+                                 internal::Object** values, int length);
   internal::Object** implicit_args_;
   internal::Object** values_;
   int length_;
-  int is_construct_call_;
```

Comment:
Yeah, removing this field will definitely break addons compiled for the existing versions of node v6. Maybe the changes can be patched up to duplicate information, so that `is_construct_call_` is always populated with a correct value? (I can look into that myself but I wouldn’t want to make any promises about time.)